app/controllers/events_controller.rb

```diff
@@ -13,6 +13,17 @@ def create
   		redirect_to @event
 	end
 
+	def edit
+    @event = Event.find(params[:id])
+  end
+
+  def update
+    @event = Event.find(params[:id])
+    @event.update(event_params)
+
```

Comment:
I would suggest to delete the extra blank line 23

Reply:
Fixed with rubocop e74418477e56ceb387e855a1c9b90138b9f90713